third_party/terraform/resources/resource_google_folder.go

```diff
@@ -63,7 +63,13 @@ func resourceGoogleFolderCreate(d *schema.ResourceData, meta interface{}) error
 		return fmt.Errorf("Error creating folder '%s' in '%s': %s", displayName, parent, err)
 	}
 
-	err = resourceManagerV2Beta1OperationWait(config.clientResourceManager, op, "creating folder")
+  opV1 := make(map[string]interface{})
```

Comment:
Watch out with spacing, these need to be tabs. See https://travis-ci.org/terraform-providers/terraform-provider-google/builds/499455964?utm_source=github_status&utm_medium=notification

Files with `.go` skip gofmt/goimports, `go.erb` have it ran which is why this can happen (and that's on purpose- using it for all `.go` files would double our compilation times)